packages/lab/src/Canvas/Toolbar/Toolbar.tsx

```diff
@@ -61,7 +64,11 @@ export const HvCanvasToolbar = (props: HvCanvasToolbarProps) => {
     <div className={cx(classes.root, className)} {...others}>
       <div className={classes.back}>
         {backButton ?? (
-          <HvIconButton title={labels.back} variant="primaryGhost">
+          <HvIconButton
+            title={labels.back}
+            variant="primaryGhost"
+            onClick={onClickBack}
```

Comment:
The component was just added, and this is already missing props 😝 

According to our guidelines, we should be calling this `onBackClick` instead:

> call handlers `on<Action>` or `on<Element><Action>` for sub-elements

We wouldn't need to name anything if we just did `backButton?: React.ReactNode` though 👀

Reply:
Actually... users shouldn't be using `onClick`, but `component={Link}` with `to={backRoute}`!

Reply:
I'll take a look a this

Reply:
I removed the click event and added `backButtonProps`: this way users can provide any prop they wish